.github/workflows/build.yml

```diff
@@ -0,0 +1,166 @@
+name: Grafana dashboards push checks
+
+on: pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
according to @atymchuk all actions should be from percona-platform, probably for the security. If that works now we need to restrict it in settings.

If there missing actions, we could fork it.

Reply:
@denisok Thanks, fixed everywhere